gen3-integration-tests/gen3_ci/jenkins_jobs/ci-only-generate-api-keys.groovy

```diff
@@ -0,0 +1,147 @@
+      valueFrom:
+        secretKeyRef:
+          name: jenkins-secret
+          key: aws_access_key_id
+    - name: AWS_SECRET_ACCESS_KEY
+      valueFrom:
+        secretKeyRef:
+          name: jenkins-secret
+          key: aws_secret_access_key
+  serviceAccount: jenkins-service
+  serviceAccountName: jenkins-service
+'''
+            defaultContainer 'shell'
+        }
+    }
+    stages {
+        stage('Clean old workspace') {
+            steps {
+                cleanWs()
+            }
+        }
+        stage('Initial setup') {
+            steps {
+                // cloud-automation
+                checkout([
+                  $class: 'GitSCM',
+                  branches: [[name: 'refs/heads/master']],
+                  doGenerateSubmoduleConfigurations: false,
+                  extensions: [[$class: 'RelativeTargetDirectory', relativeTargetDir: 'cloud-automation']],
+                  submoduleCfg: [],
+                  userRemoteConfigs: [[credentialsId: 'PlanXCyborgUserJenkins', url: 'https://github.com/uc-cdis/cloud-automation.git']]
+                ])
+            }
+        }
+        stage('Generate API Key') {
+            steps {
+                dir("ci-only-generate-api-key") {
+                    script {
+                        sh '''#!/bin/bash +x
+                            set -e
+                            export GEN3_HOME=\$WORKSPACE/cloud-automation
+                            export KUBECTL_NAMESPACE=\${NAMESPACE}
+                            source $GEN3_HOME/gen3/gen3setup.sh
+
+                            echo "creating main_account api key for \$NAMESPACE"
+                            gen3 api api-key main@example.org > \${NAMESPACE}_main_account.json
+
+                            echo "creating indexing_account for \$NAMESPACE"
+                            gen3 api api-key indexing@example.org > \${NAMESPACE}_indexing_account.json
+
+                            echo "creating auxAcct1 for \$NAMESPACE"
+                            gen3 api api-key dummy-one@example.org > \${NAMESPACE}_dummy_one.json
+
+                            echo "creating auxAcct2 for \$NAMESPACE"
+                            gen3 api api-key smarty-two@example.org > \${NAMESPACE}_smarty_two.json
+
+                            echo "creating user0 for \$NAMESPACE"
+                            gen3 api api-key user0@example.org > \${NAMESPACE}_user0_account.json
+
+                            echo "creating user1 for \$NAMESPACE"
+                            gen3 api api-key user1@example.org > \${NAMESPACE}_user1_account.json
+
+                            echo "creating user2 for \$NAMESPACE"
+                            gen3 api api-key user2@example.org > \${NAMESPACE}_user2_account.json
+                        '''
+                    }
+                }
+            }
+        }
+    }
+    post {
+        always {
+            archiveArtifacts artifacts: 'ci-only-generate-api-key/*.json'
```

Comment:
Are we creating a new file for gen3-integration-tests/gen3_ci/jenkins_jobs/ci-only-generate-api-keys.groovy? If yes, we would also need to change ci-only-generate-api-key to ci-only-generate-api-keys here